### PR TITLE
Feature/updates

### PR DESCRIPTION
**Pull request by iruzevic**
- Removing deprecated modules
- Fixing apiFetch issues
- updating overrides style

### CODE REVIEW

File: webpack/externals.js [hunk 1]
```diff
@@ -4,6 +4,17 @@
...
  */
 
+/**
+ * Convert any string to camelcase.
+ *
+ * @param string str String to convert
+ *
+ * @since 2.0.0
+ */
+function camalize(str) {
```
Comment (tihomirselak) [inline webpack/externals.js]: Is it intentional that is Camalize and not Camelize ?

Comment (iruzevic) [reply webpack/externals.js]: fixed

Comment (dingo-d) [inline webpack/externals.js]: The comment left by @tihomirselak is a valid one. `camalize` sounds wrong. And there are packages that are named `camelize` already.

File: webpack/externals.js [hunk 2]
```diff
@@ -4,6 +4,17 @@
...
 
+/**
+ * Convert any string to camelcase.
+ *
+ * @param string str String to convert
+ *
+ * @since 2.0.0
+ */
+function toCamelcase(str) {
+  return str.toLowerCase().replace(/[^a-zA-Z0-9]+(.)/g, (m, chr) => chr.toUpperCase());
```
Comment (tihomirselak) [inline webpack/externals.js]: toLowerCase why not toCamelCase

File: webpack/externals.js [hunk 3]
```diff
@@ -4,6 +4,17 @@
...
+ * Convert any string to camelcase.
+ *
+ * @param string str String to convert
+ *
+ * @since 2.0.0
+ */
+function camalize(str) {
+  return str.toLowerCase().replace(/[^a-zA-Z0-9]+(.)/g, (m, chr) => chr.toUpperCase());
+}
+
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply webpack/externals.js]: I have added the lodash

**Pull request merged**